iodata/formats/orcalog.py

```diff
@@ -33,7 +33,7 @@
 PATTERNS = ['*.out']
 
 
-@document_load_one("Orca output", ['atcoords', 'atnums', 'energy', 'moments'])
+@document_load_one("Orca output", ['atcoords', 'atnums', 'energy', 'moments', 'extra'])
```

Comment:
Just to make sure, is the "SCF ITERATIONS" always printed by default by ORCA? If not, I think, the "extra" should be listed as the "ifpresent" argument...

Reply:
I think by default they are always printed out.